lib/mix/lib/mix/tasks/new.ex

```diff
@@ -261,33 +256,29 @@ defmodule Mix.Tasks.New do
     use Mix.Project
 
     def project do
-      [app: :<%= @app %>,
-       version: "0.1.0",
-       elixir: "~> <%= @version %>",
-       build_embedded: Mix.env == :prod,
-       start_permanent: Mix.env == :prod,
-       deps: deps()]
+      [
+        app: :<%= @app %>,
+        version: "0.1.0",
+        elixir: "~> <%= @version %>",
+        build_embedded: Mix.env == :prod,
+        start_permanent: Mix.env == :prod,
+        deps: deps()
+      ]
     end
 
-    # Configuration for the OTP application
-    #
-    # Type "mix help compile.app" for more information
+    # Run "mix help compile.app" to learn about applications.
     def application do
-      # Specify extra applications you'll use from Erlang/Elixir
-  <%= @otp_app %>
+      [
+        extra_applications: [:logger]<%= @sup_app %>
+      ]
```

Comment:
This has been discussed in the past and rejected, but please conside adding a comma a the end of the last element in the keyword lists.
These lists are likely to be extended, so it makes total sense to have a comma, plus the line can be moved up in the case it want to be rearranged